.github/workflows/ci.yml

```diff
@@ -0,0 +1,27 @@
+name: CI
```

Comment:
As mentioned in #22 I'll move this over to Circle when I can view Circle output :)